pkg/cleanup/built_in_types.go

```diff
@@ -0,0 +1,95 @@
+package cleanup
+
+import (
+	sc_v1a1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	unstructured_conversion "k8s.io/apimachinery/pkg/conversion/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	api_v1 "k8s.io/client-go/pkg/api/v1"
+	apps_v1b1 "k8s.io/client-go/pkg/apis/apps/v1beta1"
+)
+
+var converter = unstructured_conversion.NewConverter(false)
+
+var MainKnownTypes = map[schema.GroupKind]SpecCleanup{
+	{Group: apps_v1b1.GroupName, Kind: "Deployment"}: deploymentCleanup,
+	{Group: api_v1.GroupName, Kind: "Service"}:       serviceCleanup,
+}
+
+var ServiceCatalogKnownTypes = map[schema.GroupKind]SpecCleanup{
+	{Group: sc_v1a1.GroupName, Kind: "Binding"}:  scBindingCleanup,
+	{Group: sc_v1a1.GroupName, Kind: "Instance"}: scInstanceCleanup,
+}
+
+func deploymentCleanup(obj *unstructured.Unstructured, actual *unstructured.Unstructured) (*unstructured.Unstructured, error) {
```

Comment:
`deploymentCleanup(obj, actual *unstructured.Unstructured)` here are in other places